types/luxon/src/datetime.d.ts

```diff
@@ -991,7 +994,7 @@ export class DateTime {
      * @example
      * DateTime.now().toLocaleString({ hour: '2-digit', minute: '2-digit', hour12: false }); //=> '11:32'
      */
-    toLocaleString(options?: LocaleOptions & DateTimeFormatOptions): string;
+    toLocaleString(formatOptions?: LocaleOptions | DateTimeFormatOptions, options?: DateTimeFormatOptions): string;
```

Comment:
I think this is wrong.

I don't have time to dig into this right now, but based on a quick look in the source code, I don't think the first argument can be of type `LocaleOptions` - which actually seems super inconvenient, and may be a bug. 

Regardless, this signature currently allows both arguments to be of type `DateTimeFormatOptions`, and that definitly seems wrong.

Reply:
You are absolutely right, on both counts. I tested this:

`dt.toLocaleString({locale: 'ru'})`

and the resulting string ignored the locale (whereas it works as expected when provided in the second argument). I've realigned the function with the implementation, and this can be further amended if Luxon implements improvements to this area.